src/js/BookReader/utils.js

```diff
@@ -159,3 +159,13 @@ export function PolyfilledCustomEvent(eventName, {bubbles = false, cancelable =
   event.initCustomEvent(eventName, bubbles, cancelable, detail);
   return event;
 }
+
+/**
+ * Promise based wait - resolves at default 500ms
+ * @param {Number} wait time in milliseconds
+ */
+export function wait(ms = 500) {
+  return new Promise((res) => {
+    setTimeout(() => res(true), ms);
+  });
```

Comment:
SUGGESTION: I love promises :)

```suggestion
  return new Promise(res => setTimeout(res, ms));
```